crates/utils/Cargo.toml

```diff
@@ -50,6 +50,7 @@ markdown-it = "0.5.1"
 totp-rs = { version = "5.0.2", features = ["gen_secret", "otpauth"] }
 ts-rs = { workspace = true, optional = true }
 enum-map = "2.6"
+async-trait = { workspace = true }
```

Comment:
Probably can be removed.

Reply:
discussed in https://github.com/LemmyNet/lemmy/pull/3742#discussion_r1282285277 for the overall test strategy